### DISCUSSION TITLE
How to add this board to Jitsi Meet

### DISCUSSION THREAD

**Discussion by onyekaLT** [OpenBoard-org/OpenBoard#1122] [Q&A]
Hi, I'm wondering if anyone could help add this board to the Jitsi Meet
let me know if it is possible
Thanks.

**Comment by letsfindaway** [comment]
You can just share the screen where OpenBoard is running, so that others see what you do on the board.

From a previous answer I gave to a similar issue:

**What you can do**

is sharing the screen where OpenBoard is running in an arbitrary video conferencing tool, That's what my wife is doing: she has a Wacom pen display connected to the laptop. BigBlueButton is running on the laptop, and the Wacom is screen-shared with the students.

**What you cannot do**

OpenBoard is not a multi-user interactive whiteboard, so students cannot interact with the board.
